container/container_unix.go

```diff
@@ -177,6 +178,10 @@ func (container *Container) HasMountFor(path string) bool {
 
 // UnmountIpcMount unmounts shm if it was mounted
 func (container *Container) UnmountIpcMount() error {
+	if runtime.GOOS == "darwin" {
```

Comment:
lets try to merge with 834d68ea0d9aa5684141748690e3a4768abb2974 (thus we won't have any diffs)